client/websocket.go

```diff
@@ -183,13 +183,16 @@ func (c *Client) handleWSMsg(msg ws.Message) error {
 				log.Printf("received call end event, closing client")
 				return errCallEnded
 			}
-		case wsEventCallRecordingState:
-			data, ok := ev.GetData()["recState"].(map[string]any)
+		case wsEventCallJobState:
+			data, ok := ev.GetData()["jobState"].(map[string]any)
 			if !ok {
 				return fmt.Errorf("invalid recording state")
 			}
 			var recState CallJobState
 			recState.FromMap(data)
+			if recState.Type != "recording" {
+				return nil
+			}
```

Comment:
Should we add a proper `WSCallJobState`? Okay to keep this for compatibility but I'd also add the new way. The higher layer should then figure out what job it is.

Reply:
Yep, that makes sense, done.